src/main/java/Kaji.java

```diff
@@ -24,7 +25,7 @@ public Kaji(String filePath) {
         cards = new CardList();
         storage = new Storage(filePath);
         try {
-            Admin admin = new Admin(storage.loadModule());
+            Admin admin = new Admin(new ModuleList(storage.loadModule()));
```

Comment:
Would the original way be better?
Admin admin = new Admin(storage.loadModule());

Then don't need to import the ModuleList class